### ISSUE TITLE
BrickPI 3 + Rapsberry PI 

### ISSUE THREAD

**Issue by Negociation** [ev3dev/ev3dev#1155]
### What are you trying to do?
Hi, i want to merge from EV3 Brick to Rapsberry Pi  , but i have some doubts , first i checked on website i olny can find a ISO for Rapsberry PI 2 , thats mean i cant work with Rpasberry PI 3 ?

- I Checked the Dexter Industries Website they only haves BrickPI 3, everone knows if BrickPI 3 as compatible with EV3DEV ?

Please i need to know that for my Final Paper on college , i probably have to buy on next week...

**Comment by jabrena** [comment]
It is compatible and running

**Comment by Negociation** [comment]
So can i run with Rapsberry PI 3 + BrickPI 3 without issues , right ?

**Comment by jabrena** [comment]
Yes, many months ago from OS layer, the support was implemented so you could develop. Other stuff is the language that want to use. If you use/like Python maybe I should add @dwalton76 in the loop. 
In relation to Java, recently we added it: 
https://github.com/ev3dev-lang-java/ev3dev-lang-java/issues/518

@mingan666 is helping us with final testing step for the combination: BrickPi3 + Raspberry Pi 3 Model B+ + Java.

**Note:** For other languages, I don´t know the real status.

Good night.

Cheers

Juan Antonio

**Comment by Negociation** [comment]
Im building a self deiving car with Convolution Neural Newtworks , so im using Python because Tensorflow.... Now im building my dataset so i can still using LEGO but when i starts the test of application I probably need more processing power i guess , that's what's I want to using the raspberry instead...

**Comment by jabrena** [comment]
Yes, definitely here, you will find good guys to collaborate.

I recommend to review this project in parallel to get some ideas.
http://www.donkeycar.com/

To manage the sensor and actuators from BrickPi3 here.

Cheers

Juan Antonio

**Comment by Negociation** [comment]
Tanks bro, i'll look later... 

For Now my robot uses:
- Lego Gyro Sensor for Fix the Foward Steering (Very Dummie but...)
- 2 Large Motors
- Color Sensor in Col-Ambient
- Ultrasonic Sensor for get the front proximity
- Infrared Sensor in IR-REMOTE to Control
- 2 Remote Controls
- Android Smartphone on Front with TCP Implementation to Recover Video

Look:
![marc](https://user-images.githubusercontent.com/32465944/44433205-be799680-a57b-11e8-9588-29195a9f62de.png)

I Want to Put front LEDs with GPIO after i replace for Rapsberry...

**Comment by mingan666** [comment]
I'm wondering what your objective is? Why do you want to use ev3dev?

For example if you want to use brickPi together with python I think you could just use the default OS from Dexter Industries. I switch to ev3dev because I'm using java which is not supported by the default OS for brickPi.

Just thinking that it might be easier to use the OS that is supported by the company that creates the BrickPi...

**Comment by Negociation** [comment]
Its a requirement port my code without change platform dude... And ev3dev with Python as a excelent choice... I checked and if i use the OS from dexter i have to change a lot of things on my code...
